rust/Cargo.toml

```diff
@@ -35,7 +35,7 @@ rust-gpu-tools = { version = "0.7", optional = true, default-features = false }
 fr32 = { version = "9.0.0", default-features = false }
 fvm4 = { package = "fvm", version = "~4.0.0-alpha.4", default-features = false }
 fvm4_shared = { package = "fvm_shared", version = "~4.0.0-alpha.4" }
-fvm3 = { package = "fvm", version = "~3.8.0", default-features = false }
+fvm3 = { package = "fvm", version = "~3.8.0", default-features = false, features = ["nv22-dev"] }
```

Comment:
should be added on fvm4? cc @Stebalien to confirm

Reply:
yes.